src/editor/core/plugins/anchors/index.js

```diff
@@ -6,27 +6,31 @@
 
 import anchor_plugin from "markdown-it-anchor";
 
-const position = {
-    false: 'push',
-    true: 'unshift'
-};
+const copyHrefToClipboard = (target, context) => {
+    const href = target.href || target.parentElement.href;
+    if (href) {
+        navigator.clipboard.writeText(href).then(r => {
+            humhub.modules.ui.status.success(context.translate('Copied'));
```

Comment:
- Maybe we should check that the `humhub.modules.ui` is available?

- Is there something needed to add the `Copied` message?

Reply:
@luke- Of course I will add this condition.
It might be better - Link has been copied to clipboard